### ISSUE TITLE
Install size of game is showing 9B and doesn't install

### ISSUE THREAD

**Issue by IvoryJam** [Heroic-Games-Launcher/HeroicGamesLauncher#3918]
### Describe the bug

When I attempt to download Titanfall 2 from Epic, it's showing an install size of 9B and obviously fails to install.

Seems to only happen on Titanfall 2 so far.

### Add logs

```text
(12:30:09) INFO:    [Legendary]:        Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(12:30:09) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(12:30:09) INFO:    [Connection]:       Connectivity: check-online
(12:30:09) INFO:    [Connection]:       Pinging external endpoints
(12:30:09) INFO:    [Gog]:              Checking for existing gog manifests
(12:30:09) INFO:    [Backend]:          DRM module staus {
  "oimompecagnajdejgnnjijobebaeigek": {
    "status": "new",
    "title": "Widevine Content Decryption Module",
    "version": "4.10.2710.0"
  }
}
(12:30:09) WARNING: [Backend]:          Failed to register protocol with OS.
(12:30:09) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(12:30:09) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(12:30:09) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/nile_config /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/nile --version
(12:30:09) INFO:    [Connection]:       Connectivity: online
(12:30:09) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:09) INFO:    [Gog]:              Checking if login is valid
(12:30:09) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:09) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(12:30:09) INFO:    [Gog]:              Updating redist, reason - new buildId
(12:30:09) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:10) INFO:    [Gog]:              Getting game metadata: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json info gog-redist --os windows
(12:30:10) INFO:    [Backend]:          
System Information:
CPU: 16x AMD Ryzen 7 2700X Eight-Core Processor
Memory: 33.59 GB (used: 6.71 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation TU106 [GeForce RTX 2070 Rev. A]
    IDs: D=1f07 V=10de SD=2173 SV=3842
    Driver: nvidia
OS: Fedora Linux 40 (Workstation Edition) (linux)

The current system is not a Steam Deck
We are running inside a Flatpak container

Software Versions:
  Heroic: 2.14.1 Flame Emperor Sabo
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.1
  Nile: 1.0.2 Jonathan Joestar

(12:30:10) ERROR:   [Gog]:              stdout = false and res.abort = false in getInstallInfo
(12:30:10) INFO:    [DownloadManager]:  Galaxy Common Redistributables  was added to the download queue.
(12:30:10) INFO:    [Gog]:              Updating GOG redistributables
(12:30:10) INFO:    [Gog]:              GOG REDIST:: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json redist --ids ISI --path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/redist/gog
(12:30:10) DEBUG:   [Gog]:              Logging to file "/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/gog-redist.log"
(12:30:10) INFO:    [Backend]:          Preventing machine to sleep
(12:30:10) INFO:    [Gog]:              Saved username to config file
(12:30:10) INFO:    [Frontend]:         Refreshing undefined Library
(12:30:10) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(12:30:10) INFO:    [Legendary]:        Refreshing library...
(12:30:10) INFO:    [Legendary]:        Refreshing Epic Games...
(12:30:10) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:10) INFO:    [Backend]:          Frontend Ready
(12:30:11) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(12:30:11) INFO:    [Gog]:              Getting GOG library
(12:30:11) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:11) INFO:    [Backend]:          Checking for current version changelog
(12:30:11) INFO:    [DownloadManager]:  gog-redist added to download manager finished.
(12:30:11) INFO:    [DownloadManager]:  gog-redist removed from download manager.
(12:30:11) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(12:30:11) INFO:    [Winetricks]:       Downloading Winetricks
(12:30:12) INFO:    [Gog]:              Saved games data
(12:30:13) INFO:    [Legendary]:        Game list updated, got 217 games & DLCs
(12:30:13) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(12:30:13) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gogdlConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(12:30:13) INFO:    [Gog]:              Found 0 game(s) to update
(12:30:13) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(12:30:14) INFO:    [WineDownloader]:   Updating wine versions info
(12:30:14) INFO:    [WineDownloader]:   Fetching upstream information...
(12:30:15) INFO:    [WineDownloader]:   wine versions updated
(12:30:15) INFO:    [Legendary]:        Found 0 games to update
(12:30:15) INFO:    [Backend]:          Starting the Download Queue
(12:30:22) INFO:    [Legendary]:        Getting more details with 'legendary info'
(12:30:22) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(12:30:22) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary info creamhorn --json --platform Windows
(12:30:28) INFO:    [Backend]:          Writing config for creamhorn
(12:30:28) DEBUG:   [Legendary]:        Using cached install info
(12:30:28) INFO:    [DownloadManager]:  Titanfall® 2: Ultimate Edition  was added to the download queue.
(12:30:28) INFO:    [Backend]:          Preventing machine to sleep
(12:30:28) DEBUG:   [Legendary]:        Using cached install info
(12:30:28) INFO:    [Legendary]:        Installing creamhorn: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install creamhorn --platform Windows --base-path /home/user/Games/Heroic --skip-dlcs -y --skip-sdl
(12:30:28) DEBUG:   [Legendary]:        Logging to file "/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/creamhorn.log"
(12:30:28) ERROR:   [DownloadManager]:  Downloader: Failed to get headers for https://raw.githubusercontent.com/Heroic-Games-Launcher/known-fixes/main/epic/creamhorn-epic.json. 
Error: Error: Request failed with status code 404
(12:30:29) INFO:    [Backend]:          Adding shortcuts for Titanfall® 2: Ultimate Edition
(12:30:29) INFO:    [DownloadManager]:  Finished Installation of creamhorn
(12:30:29) INFO:    [DownloadManager]:  creamhorn added to download manager finished.
(12:30:29) INFO:    [DownloadManager]:  creamhorn removed from download manager.
(12:30:29) INFO:    [Frontend]:         Refreshing legendary Library
(12:30:29) INFO:    [Legendary]:        Refreshing library...
(12:30:29) INFO:    [Legendary]:        Refreshing Epic Games...
(12:30:29) INFO:    [Backend]:          Stopping Power Saver Blocker
(12:30:29) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/user/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(12:30:30) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Titanfall 2 Ultimate Edition
(12:30:30) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Titanfall 2 Ultimate Edition
(12:30:30) DEBUG:   [Legendary]:        Using cached install info
(12:30:30) DEBUG:   [Legendary]:        Using cached install info
(12:30:30) ERROR:   [Legendary]:        Error: Request failed with status code 403
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:322:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1408:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(12:30:31) INFO:    [Legendary]:        Game list updated, got 217 games & DLCs
(12:30:31) ERROR:   [Legendary]:        Error: Request failed with status code 403
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:322:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1408:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(12:30:31) ERROR:   [Legendary]:        Error Getting Info from Epic API
(12:30:31) DEBUG:   [Legendary]:        Using cached install info
(12:30:31) DEBUG:   [Legendary]:        Using cached install info
(12:30:31) ERROR:   [Legendary]:        Error: Request failed with status code 403
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:322:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1408:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(12:30:31) ERROR:   [Legendary]:        Error: Request failed with status code 403
    at createError (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/app/bin/heroic/resources/app.asar/node_modules/axios/lib/adapters/http.js:322:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1408:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(12:30:31) ERROR:   [Legendary]:        Error Getting Info from Epic API
(12:31:01) DEBUG:   [Legendary]:        Using cached install info
(12:31:01) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
```

### Steps to reproduce

1. Go to Titanfall
2. Attempt to download

### Expected behavior

Install and download size are accurate

### Screenshots

![image](https://github.com/user-attachments/assets/3a269e46-45c4-494e-a5dd-27bf9f2f0211)

### Heroic Version

Latest Stable (Flatpak)

### System Information

OS: Fedora Linux 40

Software Versions:
  Heroic: 2.14.1 Flame Emperor Sabo
  Legendary: 0.20.34 Direct Intervention
  gogdl: 1.0.1
  Nile: 1.0.2 Jonathan Joestar

**Comment by arielj** [comment]
can you check if this is solved in heroic 2.15.0?

**Comment by IvoryJam** [comment]
Confirmed this works now as expected. I'm on 2.15.1 now.
